changelog/LATEST.md

```diff
@@ -3,19 +3,60 @@ View all [changelogs](https://github.com/Divine-Journey-2/Divine-Journey-2/tree/
 
 ## Mod Modifications:
 
-
+- Update AE2 Extended Life 0.55.27 -> 0.56.4
+- Update ConfigAnytime 1.0 -> 2.0
+- Update Startup Timer 1.0.1 -> 1.1.0
+- Update ZenUtils 1.15.3 -> 1.17.1
+- Update Crafttweaker 4.1.20.695 -> 4.1.20.698
+- Update Thaumic Augmentation 2.1.11 -> 2.1.13
+- Update Journeymap 5.7.1 -> 5.7.1p3
+- Update Enhanced P2P betterp2p_plusplus 1.2.3 -> betterp2p 1.2.3
 
 ## Bugfixes:
 
-
+- Removes Natura Button -> Plank recipes from the Thermal Expansion Sawmill, as it allowed you to duplicate those planks.
+- Fixes the conversion recipe for outdated Mystical Agriculture Tiered Crystals not consuming the crystal.
+- Fixes Alchemistry Titanium Plate Dissolver recipe giving 9 times what it ought to.
+- Fixes breaking Quantum Flux Graphite Ore dropping their Graphite Dust instead of our fixed Graphite Dust.
+- Fixes the Immersive Engineering Industrial Squeezer returning a HOP Graphite Ingot instead of HOP Graphite Dust.
+- Allows Primordial Fragments to be used in the Impetus Crystal recipe, only use Primordial Fragments.
+- Fixes Mekansim Sawdust being obtainable in some specific recipes.
+- Prevents a crash when pushing an OpenBlocks Trophy with a Piston by disabling the ability to push Trophies.
+- Disables the abilty to try to teleport to a specific biome via Nature's Compass, as it frequently crashes the server.
+- Fixes a bug preventing sand from converting to Glass in the Thermal Expansion Magma Crucible.
+- Fixes Mekanism Creative Tank NBT being weird in Creative Reservoir recipe.
+- Fixes an interaction between an Immersive Engineering Drill and Solar Flux Solar Panels causing extra Solar Panels to drop.
+- Fixes a visual desync when attempting to manually use an Elemental Inscription Tool, where the durability would be reduced as if you had successfully used it.
 
 ## Balance Adjustments:
 
-
+- Adds the ability to get Ghastlings without progressing through the Twilight Forest, increases their energy cost to use in the Powered Spawner.
+- Adds the ability to get Red Hearts from the Mob Loot Fabricator via the Vanilla Hostile recipe.
+- Adds the ability to get Yellow Hearts from the Mob Loot Fabricator via the Twilight Forest boss recipe.
+- Tweaks the Mystical Agriculture Chunk recipe to include 4 Inferium Essence instead of an additional 4 Soulium Ingots.
+- Removes the slightly-more-expensive recipe for Roots Unending Bowls that is done in the Vanilla Crafting Table.
+- Reduces the cost of smelting Lead Dust into Ingots, as it was 3600 instead of the 2000 for any other added recipe.
```

Comment:
Dust -> Ingot for other ingots is 1500, not 2000. (Relic, cobalt , ardite, desh,, draconium, rupee, realmite, netherite, arlemite, and corrupted draconium ingots are 2000 though.)

Reply:
i think 2000 is correct, since thats the cost of every dust -> ingot added for this machine. unifying 1500 and 2000 is a different question